crates/rust-analyzer/src/config.rs

```diff
@@ -79,6 +79,10 @@ config_data! {
         /// Run build scripts (`build.rs`) for more precise code analysis.
         cargo_runBuildScripts |
         cargo_loadOutDirsFromCheck: bool = "true",
+        /// Advanced option, fully override the command rust-analyzer uses to
+        /// run build scripts and build procedural macros. The command should
+        /// include `--message-format=json` or a similar option.
+        cargo_runBuildScriptsCommand: Option<Vec<String>> = "null",
```

Comment:
Ah damn, this is gonna run into the same problem as we had with the lifetime elision hints I believe. So this config can't be set right now, since the json path is a property of `runBuildScripts` which isn't an object, but a bool.

Reply:
cc https://github.com/rust-lang/rust-analyzer/pull/11789/files and https://github.com/rust-lang/rust-analyzer/issues/11790

Reply:
The lifetime elision config had an underscore which translated to a dot. This config does not have an underscore.

Reply:
Ah okay never mind then